src/engine/strat_engine/backstore/metadata.rs

```diff
@@ -36,7 +36,52 @@ pub struct BDA {
     regions: mda::MDARegions,
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+enum MetadataLocation {
+    Both,
+    First,
+    Second,
+}
+
 impl BDA {
+    fn read<F>(f: &mut F) -> Result<[u8; _BDA_STATIC_HDR_SIZE]>
```

Comment:
In final copy, this and other new methods should receive header doc.

Reply:
I wasn't sure if non-public functions needed documentation, but yes I can certainly add some.